content/docs/examples/multicluster/gke/index.md

```diff
@@ -7,7 +7,8 @@ keywords: [kubernetes,multicluster]
 
 This example demonstrates how to use Istio's multicluster feature to join 2
 [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) clusters together,
-using the [multicluster installation instructions](/docs/setup/kubernetes/multicluster-install/).
+using the
+[flat-network multicluster installation instructions](/docs/setup/kubernetes/multicluster-install/vpn/).
```

Comment:
Can we kill the use of "flat" in favor of routeable?  IMO, "flat network" is very loaded/misleading and not a good choice to use for anyone familiar with networking.  My vote is "routeable pod network" or "routeable pods".  This applies to the figure on the "with VPN" doc as well.

Reply:
I agree this should be changed. The term "flat network" is most commonly used to describe a network with a single L2 broadcast domain.  That isn't close to capturing the constraint that this deployment requires.

Reply:
I would also avoid using the term "pod" as the defining characteristic of the network. Would you consider:
VPN multicluster installation instructions